framework/src/modules/chain/defaults/config.js

```diff
@@ -33,13 +33,8 @@ const defaultConfig = {
 					minimum: 1,
 					maximum: 25,
 				},
-				relayLimit: {
-					type: 'integer',
-					minimum: 1,
-					maximum: 100,
-				},
 			},
-			required: ['broadcastInterval', 'releaseLimit', 'relayLimit'],
+			required: ['broadcastInterval', 'releaseLimit'],
```

Comment:
@jondubois disregard this comment. I've noticed those tests are disabled since  https://github.com/LiskHQ/lisk-sdk/commit/ec996cbf53b407d35eaaa6474f3bdd23288f7a78 @shuse2 or @MaciejBaj , any plan to reenable them?

Reply:
I already re-enabled in my PR for 2.1 https://github.com/LiskHQ/lisk-sdk/pull/3880

Reply:
Is 2.1.0 merged into development ? In that case, please @jondubois update our feature branch.

Reply:
This PR is not merged yet =)